finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/AccessLogger.scala

```diff
@@ -0,0 +1,75 @@
+package com.twitter.finagle.buoyant.h2
+
+import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.context.RemoteInfo
+import com.twitter.logging._
+import com.twitter.util._
+import java.net.InetSocketAddress
+
+object AccessLogger {
+  def mk(
+    logFilePath: Option[String],
+    logRollPolicy: Option[String],
+    logAppend: Option[Boolean],
+    logRotateCount: Option[Int]
+  ): Option[AccessLogger] = {
+    val policy = logRollPolicy match {
+      case None => Policy.Never
+      case Some(policyName) =>
+        Try(Policy.parse(policyName)) match {
+          // Default to Never roll policy if we can't parse one from the provided string
+          case Throw(_) => Policy.Never
+          case Return(r) => r
+        }
+    }
+    val append = logAppend.getOrElse(true)
+    val logRotate = logRotateCount.getOrElse(-1)
+
+    val filter = logFilePath match {
+      case Some(path) if path != "" =>
+        val logger = LoggerFactory(
+          node = "access_io.l5d.mesh",
```

Comment:
I don't think this logger node name is appropriate for an arbitrary access logger, but if you move this into the mesh iface initializer, I think it makes much more sense.